package.json

```diff
@@ -92,6 +92,7 @@
     "react-router-dom": "^5.0.1",
     "react-sortable-hoc": "^1.10.1",
     "react-sortable-tree": "^2.6.2",
+    "reactsniffer": "github:maykongsn/reactsniffer",
```

Comment:
*issue*: We definitely don't want to point to some external repos directly. Additionally if you run that tool once, there is no need to have it installed